example_problems/hello/submissions/run_time_error/test-exitcode-2.py

```diff
@@ -0,0 +1,5 @@
+'''
+@EXPECTED_RESULTS@: RUN-ERROR
+'''
+
+exit(1)
```

Comment:
Nitpick, but could we always end files with a newline? (Also below)

Reply:
with a newline char or an actual empty line?

Reply:
Only a newline char.